trestle/core/control_reader.py

```diff
@@ -710,6 +714,8 @@ def read_implemented_requirement(
             raw_comp_name = ControlReader.simplify_name(comp_name)
             if raw_comp_name == ControlReader.simplify_name(const.SSP_MD_IMPLEMENTATION_QUESTION):
                 comp_info: ComponentImpInfo = list(raw_comp_dict[raw_comp_name].items())[0][1]
+                if context.purpose == ContextPurpose.COMPONENT and not comp_info.rules:
```

Comment:
It may be that only a few of the controls have rules associated and there is no need for concern if a control has no rules.  But I'll go ahead and add a debug msg.